open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/open-lineage-connectors/open-lineage-janus-connector/README.md

```diff
@@ -7,6 +7,78 @@
 
 The Open Lineage Janus connector allows the Open Lineages Services to connect with a JanusGraph database.
 
+The Connector can be configured to connect to either an embedded JanusGraph database or a standalone JanusGraph server.
+   
+Configuring the properties of the JanusGraph client when connecting to a server should be done following the [official documentation](https://docs.janusgraph.org/basics/configuration/)
+   
+--
+- Embedded JanusGraph server
+In order to configure the connector with an embedded JanusGraph lineageGraphConnection configuration should be configured with the following `connectorProviderClassName`:  
+
+```
+ "connectorProviderClassName" : "org.odpi.openmetadata.openconnectors.governancedaemonconnectors.openlineageconnectors.janusconnector.graph.LineageGraphConnectorProvider"       
+```
+
+Example: 
+ ```
+"lineageGraphConnection": {
+    "class": "Connection",
+    "displayName": "Lineage Graph Connection",
+    "description": "Used for storing lineage in the Open Metadata format",
+    "connectorType": {
+        "class": "ConnectorType",
+        "connectorProviderClassName": "org.odpi.openmetadata.openconnectors.governancedaemonconnectors.openlineageconnectors.janusconnector.graph.LineageGraphConnectorProvider"
+    },
+   "configurationProperties": {
+        "gremlin.graph": "org.janusgraph.core.JanusGraphFactory",
+        "storage.backend": "berkeleyje",
+        "storage.directory": "./egeria-lineage-repositories/lineageGraph/berkeley",
+        "storage.berkeleyje.lock-mode": "LockMode.READ_UNCOMMITTED",
```

Comment:
Please remove 'storage.berkeleyje.lock-mode' property, it should be not default configuration (at least not at this point) since we are not sure about the behaviour; it was added while testing for locking bug we noticed in the past.